--- ansible.orig/Makefile	2013-03-11 23:12:32.586322115 +0100
+++ ansible/Makefile	2013-03-11 23:12:59.375487971 +0100
@@ -28,7 +28,7 @@
 ASCII2MAN = @echo "ERROR: AsciiDoc 'a2x' command is not installed but is required to build $(MANPAGES)" && exit 1
 endif
 
-PYTHON=python
+PYTHON=python2
 SITELIB = $(shell $(PYTHON) -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")
 
 # VERSION file provides one place to update the software version
